keps/sig-apps/2804-consolidate-workload-controllers-status/README.md

```diff
@@ -155,14 +189,14 @@ The definitions for Progressing condition (including Failed/Complete) are simila
 
 The following table is indicating what conditions are currently available (`X`) and what conditions will be added (`A`).
 
-|                                    | Waiting | Running | Progressing | Available |  ReplicaFailure | Suspended | Failed | Complete |
-| ---------------------------------  | --------| ------- | ----------- | --------- | --------------- | --------- | ------ | -------- |
-| ReplicaSet & ReplicationController | -       | -       | -           | A         | X               | -         | -      | -        |
-| Deployment                         | -       | -       | X           | X         | X               | -         | -      | -        |
-| StatefulSet                        | -       | -       | A           | A         | -               | -         | -      | -        |
-| DaemonSet                          | -       | -       | A           | A         | -               | -         | -      | -        |
-| Job                                | A       | A       | -           | -         | -               | X         | X      | X        |
-| CronJob                            | -       | -       | -           | -         | -               | -         | -      | -        |
+|                                    | Waiting | Running | Progressing | Available | Operational |  ReplicaFailure | Suspended | Failed | Complete |
+| ---------------------------------  | --------| ------- | ----------- |-----------|-------------| --------------- | --------- | ------ | -------- |
+| ReplicaSet & ReplicationController | -       | -       | -           | -         | A           | X               | -         | -      | -        |
+| Deployment                         | -       | -       | X           | X         | A           | X               | -         | -      | -        |
+| StatefulSet                        | -       | -       | A           | -         | A           | -               | -         | -      | -        |
+| DaemonSet                          | -       | -       | A           | -         | A           | -               | -         | -      | -        |
+| Job                                | A       | A       | -           | -         | -           | -               | X         | X      | X        |
```

Comment:
Why are we introducing conditions for Job that no other workload has? It seems to go against the motivation of the KEP.

Reply:
I am sorry, but the batch part needs an overhaul. Nevertheless the mission of this KEP is to examine all existing condition and try to identify new beneficial ones. But only where applicable, so we can end up only with a single condition per workload. 

An advantage of this "bird view" of all condition is that we do not introduce any overlap and inconsistency